lib/hosts.nix

```
@@ -3,7 +3,7 @@
   nixpkgs,
   ...
 }: rec {
-  mkHost = base: system: extraModules:
+  mkHost = base: system: extraModules: domain: hostname:
```

nit;
```suggestion
  mkHost = base: system: extraModules: domain: hostName:
```

for consistency with the option name

---

Sure.